src/Microsoft.Data.Analysis/DataFrame.Arrow.cs

```diff
@@ -124,6 +124,15 @@ private static void AppendDataFrameColumnFromArrowArray(Field field, IArrowArray
                     }
                     break;
                 case ArrowTypeId.Decimal128:
+                    {
+                        Decimal128Array arrowDecimal128Array = (Decimal128Array)arrowArray;
+                        var decimal128DataFrameColumn = new DecimalDataFrameColumn(fieldName, arrowDecimal128Array.Data.Length);
+                        for (int i = 0; i < arrowDecimal128Array.Data.Length; i++)
+                        {
+                            decimal128DataFrameColumn[i] = arrowDecimal128Array.GetValue(i);
+                        }
+                    }
+                    break;
```

Comment:
This code  allows to convert from Arrow to DecimalDataFrameColumn. Could you please also add backward conversion from DecimalDataFrameColumn to Arrow? It's done in `PrimitiveDataFrameColumn.ToArrowArray(long startIndex, int numberOfRows)` method